web/src/styles/layout.css

```
@@ -6,20 +6,227 @@
+.sidebar-section.is-collapsed + .sidebar-section { border-top: none; }
+.sidebar-section-title { font-size: 10px; font-weight: 600; text-transform: uppercase; letter-spacing: 0.05em; color: var(--text-tertiary); margin: 0 0 8px; padding-left: 10px; }
+.sidebar-section-toggle { display: flex; align-items: center; gap: 6px; width: 100%; border: none; background: transparent; cursor: pointer; padding: 4px 10px; font-family: inherit; }
+.sidebar-section-toggle:hover { color: var(--text); }
 
-.sidebar-item { display: flex; align-items: center; gap: 8px; width: 100%; padding: 6px 8px; border-radius: 6px; font-size: 12px; border: none; background: transparent; cursor: pointer; transition: all 0.15s; color: var(--text-secondary); font-family: var(--font-sans); text-align: left; }
+.sidebar-item { display: flex; align-items: center; gap: 10px; width: 100%; padding: 6px 10px; border-radius: 6px; font-size: 13px; border: none; background: transparent; cursor: pointer; transition: all 0.15s; color: var(--text-secondary); font-family: var(--font-sans); text-align: left; }
 .sidebar-item:hover { background: rgba(0,0,0,0.04); color: var(--text); }
 .sidebar-item.active { background: var(--accent-bg); color: var(--text); font-weight: 600; }
-.sidebar-item-icon { width: 14px; height: 14px; flex-shrink: 0; }
-.sidebar-item-emoji { font-size: 14px; flex-shrink: 0; width: 18px; text-align: center; }
+.sidebar-item-icon { width: 16px; height: 16px; flex-shrink: 0; color: currentColor; stroke-width: 2; }
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)